testpage.html

```diff
@@ -23,13 +23,18 @@
     </style>
     <script src="point-focus.js"></script>
     <style>
-      .point-focused:focus {
-      outline: none;
-      }
+        :not([point-focused]) :focus {
```

Comment:
seems fine if you want to, but do we? I mean, this would overwrite any default to style it how you want, right? I'm ok with either.